### PR TITLE
Fix/issue#102

### PR DESCRIPTION
**Pull request by AugustoCalaca**
Fix it bug reported in [#102](https://github.com/entria/entria-fullstack/issues/102) and add more tests

### CODE REVIEW

File: packages/server/src/modules/user/mutation/UserLoginWithEmailMutation.ts [hunk 1]
```diff
@@ -19,19 +19,18 @@ export default mutationWithClientMutationId({
   mutateAndGetPayload: async ({ email, password }) => {
     const user = await UserModel.findOne({ email: email.toLowerCase() });
 
-    const defaultErrorMessage = 'Invalid password';
 
     if (!user) {
       return {
-        error: defaultErrorMessage,
+        error: 'Email not found',
```
Comment (sibelius) [inline packages/server/src/modules/user/mutation/UserLoginWithEmailMutation.ts]: You should not tell that email was not found, this is a security problem

File: packages/server/src/modules/user/mutation/__tests__/UserLoginWithEmailMutation.spec.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ it('should not login if email is not in the database', async () => {
   const result = await graphql(schema, query, rootValue, context, variables);
 
   expect(result.data.UserLoginWithEmail.token).toBe(null);
-  expect(result.data.UserLoginWithEmail.error).toBe('Invalid password');
+  expect(result.data.UserLoginWithEmail.error).toBe('Email not found');
```
Comment (sibelius) [inline packages/server/src/modules/user/mutation/__tests__/UserLoginWithEmailMutation.spec.ts]: Bad error message

File: packages/server/src/modules/user/mutation/UserLoginWithEmailMutation.ts (hunk 1 above)
Comment (AugustoCalaca) [reply packages/server/src/modules/user/mutation/UserLoginWithEmailMutation.ts]: Right

**Comment by AugustoCalaca** [comment]
@sibelius merge

**Pull request merged**